Networking/Networking/Model/StripeAccount.swift

```diff
@@ -0,0 +1,128 @@
+/// Represent a Stripe Account Entity.
+///
+public struct StripeAccount: Decodable {
+    public static let gatewayID = "woocommerce-stripe"
```

Comment:
nit: is this used anywhere?

Reply:
Yes! One example is when updating the persisted gateway accounts for a given site. We allow at most one woocommerce-payments and one woocommerce-stripe account and need this in order to distinguish between them.